spec/models/claim/geckoboard_event_spec.rb

```diff
@@ -17,4 +17,24 @@
       }).to have_been_requested
     end
   end
+
+  context "with lots of claims" do
+    let(:claims) { build_list(:claim, 520, :submitted, created_at: DateTime.now) }
+
+    it "batches the claims into groups" do
+      event = Claim::GeckoboardEvent.new(claims, event_name, :created_at)
+
+      dataset_post_stub = stub_geckoboard_dataset_update("claims.#{event_name}.test")
+
+      event.record
+
+      expect(dataset_post_stub.with { |request|
+        request_body_matches_geckoboard_data_for_claims?(request, claims[0, 500], :created_at)
```

Comment:
I had to look up the syntax for using a 2nd argument with `Array[]`. Would `claims.first(500)` and `claims.last(20)` be more expressive?

Reply:
Good call 👍